pkg/config/config_template.yaml

```diff
@@ -1940,6 +1940,13 @@ api_key:
 #   <NAMESPACE_LABEL>: <TAG_KEY>
 #   <HIGH_CARDINALITY_NAMESPACE_LABEL_NAME>: +<TAG_KEY>
 
+## @param container_env_as_tags - map - optional
```

Comment:
If there's an equivalent environment variable, we are now adding those below the `@param`. For example:

```
## @env DD_CONTAINER_ENV_AS_TAGS - format - optional
```